### ISSUE TITLE
Use the * format in Fortran READ statements

### ISSUE THREAD

**Issue by Beliavsky** [algorithm-archivists/algorithm-archive#921]
I see that there have been some Fortran codes submitted but that Fortran is not available as an option on the site. Looking at the code [interactive_euclidean.f90](https://github.com/algorithm-archivists/algorithm-archive/blob/master/contents/euclidean_algorithm/code/fortran/interactive_euclidean.f90) , I suggest that 

`READ(*, '(i10)', iostat=ioerror) temp_a, temp_b`

be replaced by

`READ(*, *, iostat=ioerror) temp_a, temp_b`

which will work even if the user enters two numbers on the same line before pressing `Enter`.

**Comment by ShadowMitia** [comment]
I'm not sure what you mean by not available on the site, right not there's only one chapter that has Fortran code : https://www.algorithm-archive.org/contents/euclidean_algorithm/euclidean_algorithm.html.

If you know Fortran feel free to open PRs about it!

**Comment by leios** [comment]
Like @ShadowMitia We only have fortran code when it is submitted by community members. The only other fortran PR is #638. I am a big fan of fortran, so I would be happy to see more. If you want to create a PR about this change, feel free to do so and it will be quickly merged (I think)

**Comment by depate** [comment]
I have to check later, but there's definitely more Fortran code in the AAA. E.g. Euler and velocity-verlet. 

Regarding the initial issue. I have to check that later as well. IIRC that `(i10)` "enforces" an integer number as input, which was kind of the intention back then.

**Comment by ShadowMitia** [comment]
@depate You're right, I'm just blind I guess...

**Comment by leios** [comment]
Yeah, I think I have become more lax on type specifiers. I find them really useful for me, but I guess if other people don't think that's the case, we can remove them
